src/Contracts/NebulaField.php

```diff
@@ -17,21 +17,24 @@ abstract class NebulaField
      * Construct the field.
      *
      * @param string $name
+     * @param string $label
      */
-    public function __construct(string $name)
+    public function __construct(string $name, string $label = null)
```

Comment:
Could you reverse the order and generate a name based on the label? ex:

```
Input::make('Name')
> label: Name
> name: name

Input::make('Order Status')
> label: Order Status
> name: order_status
```

Reply:
Sure, i'm on it.

Reply:
done 😄